src/App.js

```diff
@@ -1,27 +1,13 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```

Comment:
[OPTIONAL] Please use the arrow function instead of the simple function because you used the arrow function in every file so kindly change this function as well :slightly_smiling_face: